pages/linux/link.md

```diff
@@ -0,0 +1,7 @@
+# link
+
+> Create a link to an existing file.
```

Comment:
Does it create a symbolic link? If so, then maybe we could reword this to `Create a symbolic link`....?

Also, what about a see also `ln` if more advanced options are required?

Reply:
It doesn't create a symlink. Only hard links. `ln` seems to be much better as it has symlinks and hard links, as well as error checking.